Shopipy.Persistance/Models/BusinessType.cs

```diff
@@ -0,0 +1,10 @@
+namespace Shopipy.Persistance.Models;
+
+public enum BusinessType
+{
+    Retail,
```

Comment:
did the write anything about business types and related handling? If not I would suggest dropping this part completely as it has no value.

Reply:
business types are mentioned in the swagger, but no handling logic is provided